attic_util/util.py

```diff
@@ -1,6 +1,6 @@
 import random
 import string
-import resource
+# import resource
```

Comment:
Instead of commenting this out, you can use the `try` and catch `ImportError` pattern: https://stackoverflow.com/questions/3131217/error-handling-when-importing-modules

You can move this within `memory_usage`

Reply:
Sure, I'll work on it when I get the chance.